jwst/datamodels/schemas/pathlosscorr.schema.yaml

```diff
@@ -1,5 +1,10 @@
 type: object
 properties:
+  pathloss_pointsource2d:
```

Comment:
Why create a new 2D version of the arrays in the schema, as opposed to just converting the existing ones from 1D to 2D?

Also, because the new correction factors will presumably be sampled onto exactly the same 2D grid as the science arrays, they will then also have the same wavelength assignments per pixel, so can we get rid of the wavelength arrays from the schema?

Reply:
I'm keeping the 1-d pathloss correction arrays, the wavelength arrays refer to the 1-d correction.  The 2-d pathloss correction is already interpolated using the wavelength array, so no 2-d array is needed.